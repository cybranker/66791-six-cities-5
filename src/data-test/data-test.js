export const comment = {
  comment: `What an amazing view! The house is stunning and in an amazing location. The large glass wall had an amazing view of the river!`,
  date: `2020-10-25T13:38:44.753Z`,
  id: 1,
  rating: 3,
  user: {
    [`avatar_url`]: ``,
    id: 13,
    [`is_pro`]: false,
    name: `Zak`
  }
};
