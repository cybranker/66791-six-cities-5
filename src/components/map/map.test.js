import React from "react";
import renderer from "react-test-renderer";
import Map from "./map";
import {offers, city, currentCity} from "../../data-test/data-test";

describe(`<Map /> render`, () => {
  it (`renders correctly without currentOfferLocation`, () => {
    const tree = renderer
      .create(<Map
        offers={offers}
        currentCity={city}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it (`renders correctly with currentOfferLocation`, () => {
    const tree = renderer
      .create(<Map
        offers={offers}
        currentCity={city}
        currentOfferLocation={currentCity}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
