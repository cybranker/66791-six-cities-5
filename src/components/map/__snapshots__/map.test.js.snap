// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Map /> render renders correctly with currentOfferLocation 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;

exports[`<Map /> render renders correctly without currentOfferLocation 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;
